Source/carthage/CopyFrameworks.swift

```diff
@@ -47,7 +47,7 @@ public struct CopyFrameworksCommand: CommandType {
 private func codeSigningIdentity() -> SignalProducer<String?, CarthageError> {
 	return SignalProducer.try {
 		if codeSigningAllowed() {
-			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0 }
+			return getEnvironmentVariable("EXPANDED_CODE_SIGN_IDENTITY").map { $0.isEmpty ? nil : $0 }
```

Comment:
Ooooh that makes a lot of sense. I'm only seeing this problem when compiling for the simulator anyway (not on device or when archiving) so it seems like the correct fix :+1: